odxtools/database.py

```diff
@@ -76,7 +78,7 @@ def add_auxiliary_file(self,
 
         self.auxiliary_files[str(aux_file_name)] = aux_file_obj
 
-    def _process_xml_tree(self, root: ElementTree.Element) -> None:
+    def add_xml_tree(self, root: ElementTree.Element) -> None:
```

Comment:
Not private?

Reply:
it was useful for the unit test, so I figured to make it public...